### PR TITLE
KEYCLOAK-13759 update the upgrading guide

### PR DESCRIPTION
**Pull request by andymunro**
@hmlnarik, @pdrozd Ready for your approval.

### CODE REVIEW

File: upgrading/topics/rhsso/changes-74.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+The {project_name} server was upgraded to use EAP 7.3 as the underlying container. This change does not directly involve any specific {project_name} server functionality, but a few changes relate to the migration.
+
+==== Dependency updates
+
+The dependencies were updated to the versions used by EAP 7.3 server. For example, Red Hat Data Grid is now 7.3.5.
+
+==== Configuration changes
+
+There are a few configuration changes in the standalone(-ha).xml and domain.xml files. Follow the Upgrading the Red Hat server section to handle the migration of configuration files automatically.
```
Comment (hmlnarik) [inline upgrading/topics/rhsso/changes-74.adoc]:
```suggestion
There are a few configuration changes in the standalone(-ha).xml and domain.xml files. Follow the Upgrading the {project_name} server section to handle the migration of configuration files automatically.
```

File: upgrading/topics/rhsso/changes-74.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+==== Cross-Datacenter replication changes
+
+You will need to upgrade Red Hat Data Grid to 7.3.5. The older version may still work, but it is not tested so it is not guaranteed to work.
```
Comment (hmlnarik) [inline upgrading/topics/rhsso/changes-74.adoc]:
```suggestion 
You will need to upgrade {jdgserver_name} server to version {jdgserver_version}. The older version may still work, but it is not tested so it is not guaranteed to work.
```
Plus update https://github.com/keycloak/keycloak-documentation/blob/master/topics/templates/document-attributes-product.adoc#L133-L134 appropriately

File: upgrading/topics/rhsso/changes-74.adoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+The following changes have occurred from {project_name} 7.3 to {project_name} 7.4.
+
+=== Upgrade to EAP 7.3
+
+The {project_name} server was upgraded to use EAP 7.3 as the underlying container. This change does not directly involve any specific {project_name} server functionality, but a few changes relate to the migration.
+
+==== Dependency updates
+
+The dependencies were updated to the versions used by EAP 7.3 server. For example, Red Hat Data Grid is now 7.3.5.
```
Comment (hmlnarik) [inline upgrading/topics/rhsso/changes-74.adoc]: Sorry, this is my bad, I read it wrong originally. EAP contains infinispan, not RHDG.
```suggestion
The dependencies were updated to the versions used by EAP 7.3 server. For example, Infinispan component version is now 9.3.1.Final.
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Thanks @andymunro , few minor changes and it's good to go from my perspective

**Comment by hmlnarik** [comment]
Also the following Travis failures are relevant:
```
checkExternalLinks(org.keycloak.documentation.test.ReleaseNotesTest)  Time elapsed: 8.483 sec  <<< FAILURE!
* file:/home/travis/build/keycloak/keycloak-documentation/server_admin/target/generated-docs/master.html#authentication_flow (invalid anchor authentication_flow)
* file:/home/travis/build/keycloak/keycloak-documentation/server_admin/target/generated-docs/master.html#disabling_automatic_user_creation (invalid anchor disabling_automatic_user_creation)
```

**Comment by hmlnarik** [comment]
The failures seem to originate in release notes

**Review by hmlnarik — APPROVED** [review]

**Review by pdrozd — APPROVED** [review]

**Pull request merged**